NwPluginAPI/Enums/ServerEventType.cs

```diff
@@ -607,39 +615,39 @@ public enum ServerEventType : int
 		/// Event executed when player tries to throw projectile like grenades.
 		/// </summary>
 		/// <remarks>
-		/// Parameters: <see cref="IPlayer"/> player, <see cref="ThrowableItem"/> item, <see cref="float"/> forceAmount, <see cref="float"/> upwardsFactor, <see cref="Vector3"/> torque, <see cref="Vector3"/> velocity.
+		/// Parameters: <see cref="IPlayer"/> player, <see cref="ThrowableItem"/> item, <see cref="ThrowableItem.ProjectileSettings"/> projectileSettings
```

Comment:
Not sure if bool fullForce won't be better. You can easily get ProjectileSettings by knowing if it's full force or not, but getting an information whether it's full force is only possible by comparing all the variables (in case whether strong throw == weak throw it's not possible to determine whether it was full force or not).